doc/source/groupby.rst

```diff
@@ -938,7 +937,7 @@ that is itself a series, and possibly upcast the result to a DataFrame:
 .. ipython:: python
 
     def f(x):
-      return pd.Series([ x, x**2 ], index = ['x', 'x^2'])
+        return pd.Series([x, x**2], index=['x', 'x^2'])
```

Comment:
aren't spaces required around the `**` operator?

Reply:
Somehow it doesn't complain, I changed it anyway.